modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -258,16 +278,15 @@ protected void setUpDLAppService() throws Exception {
 		);
 
 		PowerMockito.when(
-			_dlAppService.getFileEntryByUuidAndGroupId(
-				Matchers.anyString(), Matchers.anyLong())
+			_dlAppService.getFileEntryByUuidAndGroupId("0000-1111", _GROUP_ID)
```

Comment:
@carolmariaabb, why didn't you create a constant to the `uuid` as you did for the `groupId`, for example?

Reply:
@natocesarrego I forgot this one 😅. I will create a constant and update this pull. Thanks!